src/com/jwetherell/algorithms/mathematics/Division.java

```diff
@@ -3,11 +3,22 @@
 public class Division {
 
     public static final long division(int a, int b) {
+        //Quiz Two 
```

Comment:
Every method in Division.java has new functionality to check for division by zero. The two inputs a and b, regardless of numerical variable type and method of division, always result in a/b. This code ensures that, if b = 0, then an IllegalArgumentException is thrown that a divisor cannot be zero. Hence, this combats the case of division by zero.